.circleci/config.yml

```diff
@@ -2,13 +2,14 @@ version: 2.1
 
 orbs:
   snyk: snyk/snyk@0.0.12
+  browser-tools: circleci/browser-tools@1.0.1
 
 references:
   container: &container
     docker:
-      - image: circleci/node:12.13.0-browsers
+      - image: misterio92/ci-php-node
```

Comment:
Could we use a circleci php node? 

```bash
- image: circleci/php:7.1-node-browsers
```

Source: https://circleci.com/docs/2.0/language-php/

Not saying we need to use that one because I think 7.1 is out-of-date, but something similar.

Reply:
Good idea — I'll check

Reply:
Done!